src/main/java/uc/doc/submittest/SubmitTestDocInteractor.java

```diff
@@ -35,11 +39,12 @@ public SubmitTDocResponseModel submitTestDoc(SubTDocRequestModel requestModel) {
                 requestModel.getFilePath()
         );
 
-        String docID = tDocDsGateway.saveTestDoc(dsRequestModel);
+        String docId = tDocDsGateway.saveTestDocument(dsRequestModel);
+        tDocFileAccessGateway.uploadTestDocument(dsRequestModel, docId);
```

Comment:
@N1cholasMcKee the uploadTestDocument and uploadSolutionDocument functions return boolean based on successful/unsuccessful document upload. Not sure how you'd like the error handling to be done for these use cases so I've left it out for now.

Reply:
I would like to update the DB gateways in the future to also return booleans and then implement some error handling. We can deal with this once everything is tied together though so all good for now !